doc/man7/provider-asym_cipher.pod

```diff
@@ -60,73 +60,73 @@ provider_query_operation() function
 All these "functions" have a corresponding function type definition
 named B<OSSL_{name}_fn>, and a helper function to retrieve the
 function pointer from an B<OSSL_DISPATCH> element named
-B<OSSL_get_{name}>.
-For example, the "function" OP_asym_cipher_newctx() has these:
+B<OSSL_FUNC_{name}>.
+For example, the "function" OSSL_FUNC_asym_cipher_newctx() has these:
 
- typedef void *(OSSL_OP_asym_cipher_newctx_fn)(void *provctx);
- static ossl_inline OSSL_OP_asym_cipher_newctx_fn
-     OSSL_get_OP_asym_cipher_newctx(const OSSL_DISPATCH *opf);
+ typedef void *(OSSL_FUNC_asym_cipher_newctx_fn)(void *provctx);
+ static ossl_inline OSSL_FUNC_asym_cipher_newctx_fn
+     OSSL_FUNC_asym_cipher_newctx(const OSSL_DISPATCH *opf);
```

Comment:
So, while I see the charm with having this getter named being exactly the same as the function "name", I'm wondering if it won't be confusing about, 'cause when we're talking about them in text, how will we do so and still make sure that it's understood which one we talk about?  What is the nomenclature going to be?  What extra words will we have to add as context?

Reply:
I think I should add some more detailed explanation of this duality to `evp_generic_fetch(3)`, along the lines of what I wrote in the pr description.

Also, in practice I believe that there is no risk of confusion, because people will call these function through saved function pointers anyway.

Reply:
The name fits nicely to the existing description...
```pod
=head1 NAME

openssl/core_dispatch.h
- OpenSSL provider dispatch numbers and function types
```

Reply:
> I think I should add some more detailed explanation of this duality to `evp_generic_fetch(3)`, along the lines of what I wrote in the pr description.

Or maybe `provider-base(7)` would be a better place for such general explanations? :thinking:

Reply:
> ... how will we do so and still make sure that it's understood which one we talk about? What is the nomenclature going to be?

Placing "function" in quotes as a reminder, as it is done currently (`git grep -F '"function' --  doc/`), is an option but not ideal IMO. Maybe we should use the term *dispatch function* instead?

Reply:
That does remind me that I've used the term *dispatched function*  at times...  yeah, that can work

Reply:
_dispatch function_ or _dispatched function_ are both ok. Whatever you prefer.

Reply:
No, _dispatched function_ is better.

Reply:
Still tbd...